packages/ui/src/__tests__/atoms/Icon.test.tsx

```diff
@@ -17,8 +17,23 @@ describe('Icon', () => {
     expect(icon).toHaveClass('material-symbols-outlined')
 
     expect(icon).toHaveStyle(
-      `font-variation-settings: 'FILL' 1, 'wght' 400,'GRAD' 0, 'opsz' 48;`
+      `font-variation-settings: 'FILL' 0, 'wght' 400,'GRAD' 0, 'opsz' 48;`
     )
     expect(icon).toHaveStyle(`color: ${colors.primary}`)
   })
+  it('renders correctly with fill = 1', () => {
+    render(
+      <Icon data-testid="icon" name="favorite" onClick={() => { }} $fill={1}
+      >
```

Comment:
Is it necessary to have the $fill prop here? Is it not changing when you do the fireEvent.click(icon) to 1?

Reply:
yes, because the value of  the prop $fill need to be passed, so it will render filled or not